moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectTypes.js

```diff
@@ -44,6 +44,8 @@ const ProjectSummaryProjectTypes = ({
     {}
   );
 
+  console.log(typeDict);
```

Comment:
Hi Tilly - Before you merge this, do you think it's alright to remove this console.log() call?